### PR TITLE
fix(search): search on both website & docs

### PR DESCRIPTION
**Pull request by ydcjeff**
This PR enables latest ignite version search on docs and website.
Related algolia config PR to view : https://github.com/algolia/docsearch-configs/pull/4658

Ignite config: https://github.com/algolia/docsearch-configs/blob/master/configs/ignite.json
facetFilters docs (this PR using OR filter): https://www.algolia.com/doc/api-reference/api-parameters/facetFilters/#simple-or-filter

### CODE REVIEW

**Comment by ydcjeff** [comment]
@vfdev-5 done

**Review by vfdev-5 — APPROVED** [review]
Thanks @ydcjeff

File: themes/ignite/layouts/partials/SideBar.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
   <div class="border-b lg:hidden dark:border-dark-200">{{ partial "NavBar.html" . }}</div>
   <nav class="py-3 px-1">
     {{ if eq .Section "blog" }}
-      <h5 class="font-semibold py-1 uppercase">{{ .Section }}</h5>
+      <h5 class="font-semibold py-1 uppercase">{{ .Section | title }}</h5>
```
Comment (vfdev-5) [inline themes/ignite/layouts/partials/SideBar.html:34]: what this does ?

Comment (ydcjeff) [reply themes/ignite/layouts/partials/SideBar.html:34]: It changes `blog` to `Blog` which will be used in docsearch level 0

**Review by vfdev-5 — APPROVED** [review]
LGTM

**Pull request merged**